common/auto-updater/versions-db-manager.cpp

```diff
@@ -0,0 +1,307 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2020 Intel Corporation. All Rights Reserved.
+#include <fstream>
+#include <unordered_set>
+#include <unordered_map>
+#include <algorithm>
+#include <regex>
+
+#include "json.hpp"
+#include "versions-db-manager.h"
+#include "types.h"
+
+namespace rs2
+{
+    using json = nlohmann::json;
+
+    bool versions_db_manager::query_versions(const std::string &dev_name, component_part_type cp, const update_policy_type up, version& out_version)
```

Comment:
Please use full names for maintainability instead of cp, up

Reply:
Done